src/guide/conditionals.md

```diff
@@ -6,77 +6,89 @@ title: Conditionals
 
 Code can be executed conditionally using `if` expressions.
 
-### Single-sided `if`
+### `if`-`else`
 
-The `if` expression can be used with no `else` in order to execute side effects:
+An `if` expression with an `else` clause evaluates the given condition; if the condition is `true` it executes and evaluates to the expression in the `if` branch, and if the condition is `false` it executes and evaluates to the expression in the `else` branch. Both branches must evaluate to the same type.
 
 ```grain
 module Main
 
 let expr = true
-if (expr) print("Single-sided conditional")
+if (expr) print("true branch") else print("false branch")
 ```
 
+`if` is an expression, and therefore can be used like any other expression; in that way, `if` can be used like a ternary operator from other languages. For example, we can assign an `if` expression to a binding:
+
 ```grain
 module Main
 
 let expr = true
-if (expr) {
-  print("Single-sided conditional")
-  print("with multiple statements")
-}
-```
+let result = if (expr) "true branch" else "false branch"
 
-An `if` with no `else` clause always evaluates to `Void`.
-
-### Double-sided `if`
+print(result) // true branch
+```
 
-An `if` expression with an `else` clause executes the code in the `if` branch and evaluates to the last expression in the branch if the condition evaluates to `true`, and vice versa if the condition evaluates to `false`. Both branches must typecheck to the same type.
+Now is a great opportunity to introduce block expressions: to write a block expression simply surround several expressions in curly braces. When entering a block, each of the expressions in the block will be executed. The block expression as a whole will evaluate to its last expression's value. For example:
```

Comment:
I understand how this has ended up here, but I think it would be better if we moved this into the `basics.md` file and maybe just linked back somewhere, like in the next paragraph where you specify they are used together.